tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -1435,3 +1440,47 @@ def create_env_with_multiple_svc_and_lb(client, scale_svc, scale_lb,
     assert lb_service.state == "inactive"
 
     return env, services, lb_service
+
+
+def wait_for_config_propagation(super_client, lb, host_maps, timeout=30):
+    for host_map in host_maps:
+        uri = 'delegate:///?lbId={}&hostMapId={}'.\
+            format(get_plain_id(super_client, lb),
+                   get_plain_id(super_client, host_map))
+        agents = super_client.list_agent(uri=uri)
+        assert len(agents) == 1
+        agent = agents[0]
+        assert agent is not None
+        print agent
+        print agent.id
+        item = None
+        for config_items in agent.configItemStatuses():
```

Comment:
this code can be refactored to its own method as you repeat it on 1469 line again